src/goto-symex/goto_symex.h

```diff
@@ -125,6 +115,16 @@ class goto_symext
     const get_goto_functiont &get_goto_function,
     symbol_tablet &new_symbol_table);
 
+  /// Performs symbolic execution using a state and equation that have
+  /// already been used to symex part of the program. The state is not
+  /// re-initialized; instead, symbolic execution resumes from the program
+  /// counter of the saved state.
+  virtual void resume_symex_from_saved_state(
+    const get_goto_functiont &get_goto_function,
+    const statet &saved_state,
+    symex_target_equationt *const saved_equation,
+    symbol_tablet &new_symbol_table);
```

Comment:
Why has this moved (it just changed in one parameter, if I got this right)?

Reply:
Yes, I only moved it because a function was inserted splitting the two overrides apart, when surely they belong together to help the reader.